glotaran/builtin/models/kinetic_image/kinetic_image_matrix.py

```diff
@@ -29,9 +29,9 @@ def kinetic_matrix(
         raise Exception(
             f'No initial concentration specified in dataset "{dataset_descriptor.label}"'
         )
-    initial_concentration = dataset_descriptor.initial_concentration.normalized(dataset_descriptor)
+    initial_concentration = dataset_descriptor.initial_concentration.normalized()
 
-    for k_matrix in k_matrices:
+    for i, k_matrix in enumerate(k_matrices):
```

Comment:
```suggestion
    for k_matrix_index , k_matrix in enumerate(k_matrices):
```
That LGTM won't complain about the use of `i` in nested for loops and it is more apparent which `i` does what.

Reply:
Added in commit "Avoid using same variable name in nested for loops"